app/setting.py

```diff
@@ -3,7 +3,7 @@
 
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 # TODO: Try not hardcode the secret key
-SECRET_KEY='A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
+SECRET_KEY= os.environ.get('SECRET_KEY')
```

Comment:
Add a space between '='
```suggestion
SECRET_KEY = os.environ.get('SECRET_KEY')
```